test/addTwo.spec.ts

```diff
@@ -0,0 +1,7 @@
+import * as path from "path";
+import {getWasmInstance} from "./utils/utils";
+
+test('it should add two numbers', () => {
+    const instance:WebAssembly.Instance = getWasmInstance(path.join(__dirname, 'addTwo.tbs'), path.join(__dirname, 'bin/addTwo.wasm'));
```

Comment:
Of course we are generating temporary files but I recommend "bin" for all compiler output, whether its temporary or not. also I need to keep empty "bin" folder in git to avoid creating output folder from test spec files. in git "bin" is more suitable than "tmp".